compute/cmdline/pom.xml

```diff
@@ -30,8 +30,8 @@ limitations under the License.
   </parent>
 
   <properties>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Why are we bumping these to 11?

Reply:
It's supported.  I had to fix stuff anyway, and it seemed like a reasonable thing to do.

Reply:
Reverted - PTAL